### ISSUE TITLE
Dropdown: Filtering is showing as itallics not bold

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#4141]
**Describe the bug**
Noticed an example is behaving strangely.

**To Reproduce**
1. Go to https://master-enterprise.demo.design.infor.com/components/multiselect/test-reload-typeahead.html
1. Click on the label and type 'a'
1. See that it shows the highlighted character is itallics but https://master-enterprise.demo.design.infor.com/components/dropdown/example-ajax.html seems to show bold

**Expected behavior**
Should show visually as bold. Either change it to span internally or add css for making it an i

**Version**
- ids-enterprise: [e.g. v4.30]

**Platform**
 - Chrome/ Any

**Additional context**
Discovered in team chat around ajax functionality for single select dropdown

**Comment by janahintal** [comment]
This issue is now resolved. Thank you. :) 

![image](https://user-images.githubusercontent.com/45757016/90006517-a0a71e00-dccb-11ea-9d4c-d9868bd0c7b4.png)

![image](https://user-images.githubusercontent.com/45757016/90006553-b0befd80-dccb-11ea-97a5-ccc1257e586f.png)
